ingest/validation/validate_metadata.py

```diff
@@ -897,17 +897,15 @@ def collect_jsonschema_errors(metadata, convention, bq_json=None):
         return False
 
 
-def record_issue(errfile, warnfile, issue_type, msg):
+def record_issue(issue_type, msg):
     """Print issue to console with coloring and
-    writes issues to appropriate issue file
+    send errors to logger
     """
 
     if issue_type == "error":
-        errfile.write(msg + "\n")
-        dev_logger.error(msg)
```

Comment:
Is there any reason why we wouldn't want to log this in the dev logger?

Reply:
Sending an error to the user_logger will automatically also send it to the dev logger so we're covered.